### ISSUE TITLE
Composer issue

### ISSUE THREAD

**Issue by TheTechnoMan** [pingpong-labs/modules#182]
Hey, I have this issue:

    - Installation request for pingpong/modules 2.1.*@dev -> satisfiable by pingpong/modules[2.1.x-dev].
    - pingpong/modules 2.1.x-dev requires pingpong/generators 2.1.*@dev -> no matching package found.

Is there something wrong on your side? I want to install modules for laravel 5.1

Thank you!

**Comment by gravitano** [comment]
Version 2.1.*@dev is not stable. You need to require `pingpong/generators:2.1.*@dev` manually.

**Comment by TheTechnoMan** [comment]
Now I get:

  Problem 1
    - Installation request for pingpong/modules 2.1.*@dev -> satisfiable by pingpong/modules[2.1.x-dev].
    - pingpong/modules 2.1.x-dev requires pingpong/support 2.1.*@dev -> no matching package found.
  Problem 2
    - Installation request for pingpong/generators 2.1.*@dev -> satisfiable by pingpong/generators[2.1.x-dev].
    - pingpong/generators 2.1.x-dev requires pingpong/support 2.1.*@dev -> no matching package found.


I required

´´´
	    "pingpong/modules": "2.1.*@dev",
	    "pingpong/generators": "2.1.*@dev"
´´´

Am I doing something wrong? Thank you very much!

**Comment by gravitano** [comment]
Yes, that's the correct way for requiring unstable package.

**Comment by TheTechnoMan** [comment]
Alright. Just one more question, did something change in regards of fetching a config? It worked fine before, but afte rthe update it is unable to fetch the config from a module.

**Comment by TheTechnoMan** [comment]
I would really appreciate your help as I can't continue developing with this problem.

Thank you!

**Comment by TheTechnoMan** [comment]
Alright, there is clearly something wrong. I just did a clean Laravel 5.1 install and created a test module. When I then go to the module's start.php and add a ``die('Test');`` it ignores it. So it is not loaading the modules for some reason. Please take a look at that.

Thanks!

**Comment by TheTechnoMan** [comment]
The issue was that the config file was not published. You might want add a check for that.

**Comment by nWidart** [comment]
The config should not be published to work though. It should read from both locations.

**Comment by TheTechnoMan** [comment]
Yeah, that is was I would expect. Seems like a bug to me, I will therefore reopen the issue. I can fetch the module's config fine, the only problem was that they did not work until the main package config got published.
